internal/common/profiling/http.go

```diff
@@ -0,0 +1,24 @@
+package profiling
+
+import (
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
Importing the internal/common/profiling package causes pprof endpoints to be registered.